numba/tests/test_array_methods.py

```diff
@@ -170,6 +179,12 @@ def array_sum_const_axis_neg_one(a, axis):
     # "const_multi" variant would raise errors
     return a.sum(axis=-1)
 
+def array_mean(a):
```

Comment:
not sure these are supposed to be here either?

Reply:
you're too fast, I was just reverting those. I think I got on the wrong side of PyCharm smart checkout, and it brought those from my branch working on `np.mean`. I think I cleaned it now.